app/graphql/types/enum_helpers.rb

```diff
@@ -3,25 +3,31 @@ module EnumHelpers
     def self.enum_from_model(model, enum_name, graphql_enum_name: nil)
       enum_values = model.send(enum_name.to_s.pluralize).keys
 
-      Class.new(Types::BaseEnum) do
-        graphql_name(graphql_enum_name || "#{model.name}#{enum_name.to_s.camelize}Enum")
+      graphql_class_name = graphql_enum_name || "#{model.name}#{enum_name.to_s.camelize}Enum"
```

Comment:
@Mahaveer1141  - this code seems to be duplicated a bit, as we scale this out and add more helpers, perhaps we can add a method to do this in once place?